astor/code_gen.py

```diff
@@ -19,15 +19,18 @@
 
 import ast
 import sys
+import gc
 
 from .op_util import get_op_symbol, get_op_precedence, Precedence
 from .node_util import ExplicitNodeVisitor
-from .string_repr import pretty_string
-from .source_repr import pretty_source
+from .prettifier import StringLiteral, beautify
 
+###################################################################
+#               Main interface
+###################################################################
 
 def to_source(node, indent_with=' ' * 4, add_line_information=False,
-              pretty_string=pretty_string, pretty_source=pretty_source):
```

Comment:
So, unless you know of a major user using them, we should probably not support them, since anybody using them was pulling from git rather than pypi anyway.